java/src/jmri/jmrix/openlcb/OlcbProgrammer.java

```diff
@@ -25,8 +25,8 @@
  * Depending on which type you have, only certain modes can be set. Valid modes
  * are specified by the class static constants.
  * <P>
- * You get a Programmer object from a {@link jmri.ProgrammerManager}, which in turn
- * can be located from the {@link jmri.InstanceManager}.
+ * You get a Programmer object from a ProgrammerManager, which in turn
```

Comment:
Curious why you took out the links to the classes.  I generally like those, as it makes it easy to click to the class being referenced.